addons/account/controllers/portal.py

```diff
@@ -65,6 +65,9 @@ def portal_my_invoices(self, page=1, date_begin=None, date_end=None, sortby=None
             'invoices': {'label': _('Invoices'), 'domain': [('move_type', '=', ('out_invoice', 'out_refund'))]},
             'bills': {'label': _('Bills'), 'domain': [('move_type', '=', ('in_invoice', 'in_refund'))]},
         }
+
+        searchbar_inputs = self._invoice_get_searchbar_inputs()
```

Comment:
put this directly at : 'searchbar_inputs': self._invoice_get_searchbar_inputs() line 109